pysatModels/utils/match.py

```diff
@@ -16,12 +17,14 @@
 from __future__ import unicode_literals
 
 import datetime as dt
+import logging
 import numpy as np
 from os import path
-from pandas import (DateOffset, date_range)
 
+from pandas import (DateOffset, date_range)
```

Comment:
Are tuples of methods part of a style recommendation?

Reply:
Don't recall, I normally don't use them.